test/Sema/option-set-empty.swift

```diff
@@ -0,0 +1,4 @@
+struct MyOptions: OptionSet {
+    var rawValue: Int
+    static let none = MyOptions(rawValue: 0) // expected-warning {{'static let' constant inside 'MyOptions' that conforms to OptionSet produces an empty option set}}
```

Comment:
Can you verify if the fix-it is correctly applied? You need to add it at the end of this diagnostic. The format is:

```
{{column_start-column_end=<text>}}
```

Example:

```
// expected-warning {{warning_text}}{{30-34=([])}}
```

Reply:
Thanks! I'd just noticed that.

Reply:
Also, is there any way to run a specific unit test instead of the whole suite? I'm currently using `ninja check-swift`

Reply:
You can invoke `lit` directly, it lives inside the LLVM checkout

```
$LLVM_DIR/utils/lit/lit.py -sv $BUILD_DIR/swift-macosx-x84_64/test-macosx-x86_64 --filter=option-set-empty
```

Reply:
Yeah, that was just an example.

Reply:
@harlanhaskins It's saying "Test has no run line!". Sorry if I'm missing something obvious.

Reply:
Nevermind. Figured it out.